test/integration/workarea/storefront/global_e_api/receive_order_integration_test.rb

```diff
@@ -225,6 +225,22 @@ def test_with_shipping_discounts
           assert_equal 1, order.international_discount_adjustments.size
         end
 
+        def test_with_tax_subsidy_discount
+          order = create_cart
+
+          post storefront.globale_receive_order_path,
+            headers: { 'CONTENT_TYPE' => 'application/json' },
+            params: global_e_send_order_to_merchant_with_shipping_discounts_body(order: order)
+
+          assert response.ok?, "Expected 200 response"
+
+          order.reload
+
+          assert_equal "$53.03", order.duties_and_taxes_nonsubsidized.format
```

Comment:
I wouldn't use `.format` here since other people could have a non-USD default currency. Instead, test for `53.03.to_m`.

Reply:
I just copied the pattern in the file. Do you want all instances updated?

Reply:
just change the ones in your test, we can fix the rest of them at a later date.